Content/Biomes/PermafrostTempleBiome.cs

```diff
@@ -17,5 +17,14 @@ public override bool IsBiomeActive(Player player)
 		{
 			return Main.tile[(int)player.Center.X / 16, (int)player.Center.Y / 16].WallType == ModContent.WallType<AuroraBrickWall>();
 		}
+
+		public override void OnInBiome(Player player)
+		{
+			if (player == Main.LocalPlayer && Main.SmartCursorWanted)
+			{
+				Main.SmartCursorWanted_Mouse = false;
```

Comment:
we should keep track of their desired smart cursor setting and reenable it once when they leave the biome if it was on before entering

Reply:
Should be memorized now